sql/plsql/PlSqlParser.g4

```diff
@@ -4744,7 +4744,7 @@ modify_table_partition
     : MODIFY (
         PARTITION partition_name ((ADD | DROP) list_values_clause)? (ADD range_subpartition_desc)? (
             REBUILD? UNUSABLE LOCAL INDEXES
-        )?
+        )? (shrink_clause)?
```

Comment:
Please remove redundant parentheses:

```suggestion
        )? shrink_clause?
```

Reply:
Thanks for your suggestion. I have addressed it.